skins/cat17/src/app/lib/animated_scroller.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+var objectAssign = require( 'object-assign' ),
+	AnimatedScroller = {
+		fixedHeaderElements: null,
+		scrollTo: function( $element ) {
+			var offset = 0,
+				matchedElemPadding = $element.css( 'padding-top' ).match( /^(\d+)px$/ );
+			this.fixedHeaderElements.each( function ( idx, elm ) {
```

Comment:
With `_.each` it [becomes testable](https://github.com/wmde/FundraisingFrontend/blob/master/skins/cat17/src/app/tests/test_form_data_extractor.js#L18).